components/generic/ContentCard.vue

```diff
@@ -141,6 +141,10 @@
         type: Boolean,
         default: true
       },
+      noSubtitle: {
```

Comment:
Let's name this showSubtitle instead, defaulting to true.

Reply:
good idea